common/config/rush/pnpm-lock.yaml

```diff
@@ -2304,7 +2304,7 @@ packages:
     resolution: {integrity: sha512-ALYone6pm6QmwZoAgeyNksccT9Q4AWZQ6PvfwR37GT6r6FWUPguq6sUmNGSMV2Wr761oQoBxwGGa6DR5o1DC9g==}
     dependencies:
       '@types/connect': 3.4.35
-      '@types/node': 12.20.55
+      '@types/node': 18.8.0
```

Comment:
Not sure where this bump is coming from, is it ok? @jeremymeng

Reply:
It is some weird rush behavior that we haven't been able to figure out the cause. `rush update --full` make it v12, which seems better as we have @types/node v12 in our package.json.  However, `rush update` would just bump it to v18.  Will have to live with it now unless we find out how to fix it.

Reply:
Yeah, odd. I really do wonder what's causing it.